scst-store/multicluster-setup.hbs.md

```diff
@@ -17,20 +17,27 @@ cluster to communicate with SCST - Store in the View cluster.
 > profile](#install-build-profile) step, instead of deploying the Build profile
 > again, update your deployment using `tanzu package installed update`.
 
+![Screenshot of the SCST - Store in Multi Cluster Deployment](images/multicluster-deployment.jpg)
+
 ## Prerequisites
 
 You must install the View profile. See [Install View profile](../multicluster/installing-multicluster.hbs.md#install-view).
 
 ## Procedure summary
 
-1. Copy SCST - Store CA certificate from the View cluster.
+1. Copy CA certificates from the View cluster.
+   1. Copy Metadata Store CA certificate from the View cluster.
+   2. Copy AMR Cloud Event Handler CA certificate from the View cluster.
 2. Copy SCST - Store authentication token from the View cluster.
-3. Apply the CA certificate and authentication token to the Kubernetes cluster where you intend to install the Build profile.
+   1. Copy Metadata Store authentication token from the View cluster.
+   2. Copy AMR Cloud Event Handler edit token from the View cluster.
+3. Apply the CA certificates, authentication token and cloud event handler edit token to the Kubernetes cluster where you intend to install the Build profile.
```

Comment:
Nit: change authentication token to Metadata Store authentication token (primarily because you had it on the previous step to there is no confusion on which auth token)